### DISCUSSION TITLE
Korrekturen / Ergänzungen

### DISCUSSION THREAD

**Discussion by Plakatsammlung** [maehr/diskriminierungsfreie-metadaten#4] [General]
Herzlichen Dank für Eure wertvolle Arbeit mit dem «Handbuch zur Erstellung diskriminierungsfreier Metadaten für historische Quellen und Forschungsdaten».

Eine kleine Korrektur beim Beispiel 5.4 (https://maehr.github.io/diskriminierungsfreie-metadaten/#hie-basel-hie-schweizer-boden---liste-3).Bei der Angabe «Fotoarchiv Hofmann» unter Rights scheint es sich um einen Fehler zu handeln. Es handelt sich um den Scan einer manuell hergestellten Lithografie.

& eine mögliche Ergänzung. Die Plakatsammlung der Schule für Gestaltung Basel (Plakatsammlung SfG Basel) hat eine Wikidata-ID: Q19362827

Herzlich, Philipp

**Comment by maehr** [comment]
Lieber Philipp 

Vielen herzlichen Dank für die Korrektur. @noelleschnegg und ich werden das asap korrigieren.

Beste Grüsse
Moritz

PS: Die Änderungen werden mit https://github.com/maehr/diskriminierungsfreie-metadaten/pull/3 umgesetzt.

**Comment by el-zm** [comment]
Lieber Moritz, liebe Noëlle

Auch auf diesem Weg nochmals herzlichen Dank für das wichtige Projekt und das gelungene Handbuch! 

Ich habe ebenfalls eine kleine Anmerkung zum Beispiel 5.4 (https://maehr.github.io/diskriminierungsfreie-metadaten/#hie-basel-hie-schweizer-boden---liste-3): Unter den Schlagwörtern zu diesem Bild fehlt m.E. noch der Begriff des Antisemitismus, der sich angesichts der Bilbdeschreibung mit dem Verweis auf die "überzeichneten Nasen" aufdrängt. 

Schwieriger zu korrigieren wäre meine Anmerkung zum Beispiel 5.8. (https://maehr.github.io/diskriminierungsfreie-metadaten/#plan-mit-verteilung-j%C3%BCdischer-h%C3%A4user). Die Verschlagwortung des Plans mit Verteilung jüdischer Häuser um 1398 führt nun zwar "Antisemitismus" auf, strenggenommen müsste man im Mittelalter jedoch eher (oder zumindest zusätzlich) von "Antijudaismus" sprechen - was zum eigentlichen Problem führt: Der Begriff "Antijudaismus" existiert im Schlagwortindex nicht, Antisemitismus bezeichnet eigentlich einen modernen, biologistisch statt religiös geprägten Diskurs. Wäre es Eurer Meinung nach sinnvoll, die allzu vage Verschlagwortung in der Description zu verfeinern oder würdet Ihr Euch bewusst gegen solche Begriffserklärungen entscheiden, um die Description nicht zu überfrachten?

Herzliche Grüße
Elias

**Reply by maehr** [reply]
Vielen Dank für die ausführliche Rückmeldung. @noelleschnegg und ich werden das asap anschauen und das Handbuch dementsprechend anpassen.
Beste Grüsse 
Moritz Mähr

**Reply by noelleschnegg** [reply]
Wie wäre es, wenn ich die Probleme bei der Verschlagwortung in der Schritt-für-Schritt-Anleitung unter Punkt 4.6 "Erfahrungen der Stadt.Geschichte.Basel" anmerken würde? (https://maehr.github.io/diskriminierungsfreie-metadaten/#sechster-schritt-datenwertstandards-w%C3%A4hlen)

**Reply by el-zm** [reply]
Ja, wenn sich das dort kommentieren und direkt mit den herausfordernden Fallbeispielen verküpfen lässt, wäre es an dieser Stelle sicherlich hilfreich, herzlichen Dank!

**Reply by noelleschnegg** [reply]
Vielen Dank für die rasche Rückmeldung!

**Comment by maehr** [comment]
> From: Lackner, Karin ([karin.lackner@uni-graz.at](mailto:karin.lackner@uni-graz.at)) <[karin.lackner@uni-graz.at](mailto:karin.lackner@uni-graz.at)>
> Sent: Friday, July 19, 2024 1:15:34 PM
> To: [moritz.maehr@unibas.ch](mailto:moritz.maehr@unibas.ch) <[moritz.maehr@unibas.ch](mailto:moritz.maehr@unibas.ch)>; [noelle.schnegg@unibas.ch](mailto:noelle.schnegg@unibas.ch) <[noelle.schnegg@unibas.ch](mailto:noelle.schnegg@unibas.ch)>
> Subject: Rückmeldung zu Ihrem Handbuch zur Erstellung diskriminierungsfreier Metadaten für historische Quellen
>  
> Sehr geehrte Frau Schnegg, sehr geehrter Herr Dr. Mähr,
>  
> ich bin Bibliothekarin an der Universitätsbibliothek Graz und finde Ihr Handbuch sehr interessant. Es wäre sehr wünschenswert, wenn derartige Kontextinformationen zu problematischen historischen Darstellungen bzw. Werken häufiger in den Metadaten von Onlinekatalogen und Discovery Systemen zur Sensibilisierung der Nutzer:innen angeboten würden (was angesichts des Zeitaufwands, derartige Informationen zu recherchieren und einzupflegen, wohl allerdings leider an Grenzen stößt).
>  
> Zu einigen der im Leitfaden vorgestellten Beispiele möchte ich kurz meine Überlegungen teilen – vielleicht ist die eine oder andere Anregung zur Ergänzung/Änderung dabei, die Sie ggf. in das Handbuch aufnehmen möchten.
>  
> Was mir generell aufgefallen ist, dass in den Beschreibungen keinerlei Quellen- oder Literaturhinweise gegeben werden, was in manchen Fällen sehr schade ist. Mir ist bewusst, dass Sie und weitere Historiker:innen an dem Projekt gearbeitet haben und die Informationen in den Beschreibungstexten sicherlich zuverlässig sein werden, allerdings werden Datensätze ggf. von Metasuchmaschinen etc. geharvestet, und wenn die Datensätze dadurch aus dem Zusammenhang gerissen und nicht mehr unmittelbar mit dem Projekt (in diesem Fall zur Stadtgeschichte Basel) in Verbindung gebracht werden können, ist es für die Nutzer:innen nicht mehr offensichtlich, woher bzw. von wem die Angaben stammen. Damit ergibt sich automatisch die Frage nach der Zuverlässigkeit der Informationen, insbesondere, falls diese dem eigenen (vermeintlichen) Wissensstand widersprechen sollten. Quellenangaben wären in diesen Fällen für die Nutzer:innen sicherlich hilfreich.
>  
> Bei Abb. 17 – das im Handbuch als positives Beispiel für Lebensbilder gebracht wird – habe ich mich beim Lesen unwillkürlich gefragt, weshalb es als positiv bewertet wird, wenn hier Kinder als Teil einer – wie es im Beschreibungstext heißt – nicht-kindgerechten Handlung dargestellt werden. Hier wäre meiner Meinung nach eine kurze Erläuterung (nicht im Metadatenfeld Description, sondern im Handbuch selbst) hilfreich, weshalb dieses Beispiel im Sinne des Handbuchs dennoch als positives Beispiel ausgewählt wurde (weil Kinder überhaupt dargestellt werden, die in historischen Darstellungen oft fehlen oder stark unterrepräsentiert sind?). Eventuell wäre ergänzend auch ein Hinweis darauf, dass hier eine Frau aktiv am Ritual beteiligt ist, überlegenswert – umso mehr, als bei Abb. 14 explizit darauf hingewiesen wird, dass dies dort nicht der Fall ist.
>  
> Und während ich die Kritik an der Darstellung der Dame in Abb. 16 absolut nachvollziehen kann, finde ich die Beschreibung der Keltin in Abb. 19 mit „hat keine Idealfigur“ etwas eigenartig. Wer bestimmt, was eine Idealfigur ist? Vielleicht war das, was die Dame hat, die keltische Idealfigur? Außerdem finde ich es merkwürdig, dass das erste, was über die dargestellte Dame – offenbar eine hochrangige Frau – im Beschreibungstext erwähnt wird, die Figur ist. Frauen werden auch heute noch oft auf ihr Äußeres reduziert, und der Beschreibungstext macht dasselbe. Als wäre die Figur das Allerwichtigste an der Dame. Dass das als Beispiel einer diskriminierungsfreien Beschreibung gebracht wird, finde ich befremdlich. Vielleicht könnte man den Hinweis auf die Figur – wenn er denn unbedingt sein muss – ans Ende des Beschreibungstextes stellen, denn meiner Meinung nach ist das Wichtigste an der Darstellung, dass sie eine mächtige, einflussreiche und selbstbewusste hochrangige Keltin repräsentiert, und nicht, welche Figur sie hat.
>  
> Bitte verstehen Sie meine Anmerkungen nicht als Kritik, ich finde das Projekt und die Initiative zu möglichst diskriminierungsfreien Metadaten vorbildlich und wichtig. Da das Handbuch eingangs als Living Document bezeichnet wird, wollte ich lediglich meine Überlegungen teilen, vielleicht sind ein paar Anregungen dabei, die Sie im Handbuch direkt bzw. in den Beschreibungstexten der Datensätze umsetzen möchten.
>  
> Mit freundlichen Grüßen
> Karin Lackner
>  
> ____________________________________
>  
> MMag. Karin Lackner
> Informationsdienste | Information Services
> Publikationsservices | Publication Services
>  
> Universitätsbibliothek | University Library
> Universität Graz | University of Graz
>  
> Universitätsplatz 3a, 8010 Graz
> Tel.: +43/(0)316/380 – 1435
> E-Mail: [karin.lackner@uni-graz.at](mailto:karin.lackner@uni-graz.at)
> Web: http://ub.uni-graz.at/  
>            
>  
> Bitte denken Sie an die Umwelt, bevor Sie dieses E-Mail drucken. Danke!
> Please consider the environment before printing this e-mail. Thank you!

Hinweis: Diese Mail wurde mit ausdrücklicher Erlaubnis von Frau Lackner veröffentlicht.

**Reply by maehr** [reply]
> From: Spano Roberta Flora <roberta.spano@library.ethz.ch>
> Date: Wednesday, 11 September 2024 at 16:01
> To: "moritz.maehr@unibas.ch" <moritz.maehr@unibas.ch>
> Cc: "noelle.schnegg@unibas.ch" <noelle.schnegg@unibas.ch>
> Subject: Handbuch diskriminierungsfreie Metadaten - Feedback 
> 
> Lieber Moritz
> 
> Ich hoffe, es geht dir gut. An der ETH-Bib haben sich verschiedene Leute euer Handbuch durchgelesen – vielen Dank für diese wertvolle Arbeit. 
> Es sind verschiedene Kommentare zusammengekommen, die ich euch gerne weiterleite. Sie sind von verschiedenen Personen verfasst und ich habe sie nach Beispielen sortiert. Vielleicht ist ja das ein oder andere für eine allfällige Überarbeitung hilfreich. 
> 
> -	5.1: Das Bad zu Leuk 
> o	Subject: Sexismus und sexuelle Belästigung ist m.E. falsch resp. ein Kategorienfehler (Sexismus ist ein moderner Begriff. Grundlage ist auch die Neu-Codierung der Geschlechter vgl. Claudia Honegger: Die Ordnung der Geschlechter. Die Wissenschaften vom Menschen und das Weib)
> o	Fast alle Dargestellten sind nackt, demnach macht der Satz «Darüber hinaus wurden Frauen im 16. Jahrhundert oft objektiviert und als kindlich und unschuldig stereotypisiert, was sich in der Darstellung junger, unverhüllter Körper in der Kunst widerspiegelt.» wenig Sinn, zudem ist das heute nicht anders
> o	Schlagworte sexuelle Belästigung und Erotik gleichzeitig > bräuchte es da ev. noch etwas Kontext? 
> 
> -	5.2.: Plakat zur Völkerschau, 1926
> o	Als Bibliothekarin würde man die Metadaten auf dem Plakat ebenfalls erfassen (Autopsie): Titelangaben werden genau im Wortlaut übernommen (auch wenn sie «problematisch» sind). So wie es hier gemacht wurde, wäre ein Vermerk wie «konstruierter Titel» angebracht. Zumal man ja nach Methode 1 Titelkontextualisierung vorgehen möchte (siehe 4.1.5.1.5 Begriffe)
> 
> -	5.4: Hie Basel Hie Schweizer Boden – Liste 3
> o	Hier hat man die Metadaten auf dem Plakat im Wortlaut übernommen. 
> 
> -	5.7: Basler Brandmarkeisen, 17. Jahrhundert
> o	Offensichtlich sind es zwei unterschiedliche Eisen. Im Text wird auf zwei Arten der Brandmarkung hingewiesen (Integration und Verbannung). Interessant wäre nun zu wissen, welches Eisen für welche Markierung war. Falls man dies nicht weiss, wäre auch das eine gute Information. 
> 
> -	5.8: Plan mit Verteilung jüdischer Häuser
> o	Date: 1398 --? Der Plan wurde bestimmt erst 2010 erstellt. Die Datierung datiert in der Regel das vorliegende Dokument. 1398 ist die «inhaltliche Datierung», der Plan rekonstruiert die Situation des Jahres 1398.
> 
> -	5.9.2.3.: Die Keltin 
> o	Hinweis auf die nicht vorhandene «Idealfigur», braucht es da nicht eine Kontextualisierung, ist das nicht aus heutiger Sicht gesprochen? 
> 
> 
> Viele liebe Grüsse, 
> Roberta 
> 
> ETH Zürich, ETH-Bibliothek, Roberta Spano, Sammlungen und Archive, Rämistrasse 101, 8092 Zürich, +41 44 632 36 79, anwesend: Mo-Do, [roberta.spano@library.ethz.ch](mailto:roberta.spano@library.ethz.ch), [www.library.ethz.ch](http://www.library.ethz.ch/), [Sammlung wissenschaftlicher Instrumente und Lehrmittel](https://ethz.ch/de/campus/erleben/lernen-und-arbeiten/sammlungen-und-archive/sammlung-wissenschaftliche-instrumente-und-lehrmittel.html)

Hinweis: Diese Mail wurde mit ausdrücklicher Erlaubnis von Frau Spano et al. veröffentlicht.

**Comment by noelleschnegg** [comment]
Liebes OpenGender-Team
Liebe Karin Aleksander

Mein Name ist Noëlle Schnegg und ich arbeite zusammen mit Levyn Bürki und Moritz Mähr an einer Neuauflage eines [Handbuchs](https://maehr.github.io/diskriminierungsfreie-metadaten/) zum sensiblen Umgang mit Metadaten bei historischen Forschungsdaten. Das Handbuch entstand im Rahmen des geschichtswissenschaftlichen Forschungsprojekts Stadt.Geschichte.Basel und legt einen besonderen Fokus auf Quellen, die unterschiedliche Formen von Diskriminierung enthalten. Dazu haben wir den GenderOpen-Schlagwortindex verwendet und dabei auch ein Mapping zur GND erstellt.
Beim Arbeiten mit dem GenderOpen-Schlagwortindex haben sich bei uns einige Fragen gestellt, die wir laufend notiert haben und gerne mit Ihnen teilen möchten. Der OpenGender-Schlagwortindex war uns ein äusserst wertvolles Werkzeug. Deshalb sind wir insbesondere auch an einem Austausch interessiert, wie der Index allenfalls weitergeführt werden könnte.

Über einen Austausch und Einblicke in die Entscheidungsprozesse hinter dem Schlagwortindex würden wir uns sehr freuen. Nachfolgend in drei Punkten einige konkrete Fragen, über die wir regelmässig gestolpert sind, und wo wir uns über Antworten sehr freuen würden:
1.	Inkonsistenzen btr. Sprache: Uns fiel auf, dass der Index eine Vielzahl von Anglizismen enthält -- darunter auch solche, für welche durchaus gebräuchliche deutsche Begriffe existieren würden. Da es sich um einen deutschen Index handelt, interessiert uns natürlich, wie in den einzelnen Fällen entschieden wurde.
2.	Inkonsistenzen btr. Plural/Singular: Während Begriffe im Plural aufgeführt sind (z. B. Frauen in Führungspositionen oder Schwule), stehen andere im singular (z. B. Familienform). Wurden diese Entscheidungen jeweils bewusst getroffen?
3.	Unklare Begriffsdefinitionen und Angaben zur Begriffsauswahl: Beim Verwenden der Schlagworte -- und ganz besonders beim Mapping zur GND -- haben uns mehrfach klare Begriffsdefinition der OpenGender-Schlagworte gefehlt. 
1.	Besonders nützlich wäre dies bei sehr ähnlichen Begriffen. So stellten wir uns bei Female Genital Cutting und Female Genital Mutilation etwa die Frage, ob wir die zwei Begriffe als Quasisynonyme verwenden und mappen können, oder ob Sie hier auf eine Unterscheidung wert legen -- und falls ja, weshalb?
2.	Auch bei sehr spezifischen Begriffen wie Schwarze Deutsche wäre eine Begriffsdefinition sehr hilfreich. Denn ohne den entsprechenden Kontext ist in diesem speziellen Fall beispielsweise nicht ersichtlich, ob es sich hier um einen bestimmten Singular oder um einen unbestimmten Plural handelt.
4.	Erweiterung: Gerne würden wir die Aufnahme der Begriffe Sexualisierung und Male gaze vorschlagen, welche uns zur Verschlagwortung unserer Bildquellen gefehlt haben. Ist die Möglichkeit einer Erweiterung des Index angedacht?
Wir freuen uns sehr auf Ihre Rückmeldung und einen Austausch (sehr gerne auch per Videocall).
PS: Wir waren uns nicht ganz sicher, ob die Projekt-Email fünf Jahre nach Projektende noch betreut wird. Andrea Gruber von der Österreichischen Nationalbibliothek hat uns deshalb letzte Woche an der DHd geraten, die Email zusätzlich noch an Karin Aleksander weiterzuleiten.

Mit freundlichen Grüssen
Noëlle Schnegg

**Comment by maehr** [comment]
Diese Diskussion wird geschlossen, Anregungen zur zweiten, erweiterten Auflage werden unter https://github.com/maehr/diskriminierungsfreie-metadaten/discussions/34 entgegengenommen.
